tests/integration_test.rs

```diff
@@ -0,0 +1,80 @@
+//! Tests for `get_user` and `post_user` methods
+//! Note: the endpoints for this lib are HARD CODED. only to be used with `https://jsonplaceholder.typicode.com/api/v1/users`
+
+extern crate actix_basic;
+
+use tokio;
+use serde_json::json;
+
+const URL: &str = "https://jsonplaceholder.typicode.com/api/v1/users";
+
+#[tokio::main]
+#[test]
+async fn it_gets_users() -> Result<(),Box<dyn std::error::Error>>{
+
+    let address_check =  actix_basic::Address {
+        street: String::from("Douglas Extension"),
+        suite: String::from("Suite 847"),
+        city: String::from("McKenziehaven"),
+        zipcode: String::from("59590-4157"),
+        geo: actix_basic::Geo {
+            lat: String::from("-68.6102"),
+            lng: String::from("-47.0653"),
+        },
+    };
+
+    let users: Vec<actix_basic::User> = actix_basic::get_users(URL).await?;
```

Comment:
For an external testing folder, you should be hitting the actual endpoint, not the handler.